packages/tempus/src/Tempus_StepperHHTAlpha_impl.hpp

```diff
@@ -488,17 +488,18 @@ void StepperHHTAlpha<Scalar>::describe(
    Teuchos::FancyOStream               &out,
    const Teuchos::EVerbosityLevel      verbLevel) const
 {
+  auto l_out = Teuchos::fancyOStream( out.getOStream() );
+  Teuchos::OSTab ostab(*l_out, 2, this->description());
+  l_out->setOutputToRootOnly(0);
 
 #ifdef VERBOSE_DEBUG_OUTPUT
   *out_ << "DEBUG: " << __PRETTY_FUNCTION__ << "\n";
 #endif
```

Comment:
Is this needed anymore?

Reply:
Yeah, there are many things in HHTAlpha and Newmark* classes that are consistent with other Steppers.  The current plan to replace these with a Generalized-Alpha stepper, so I have been putting off changes in HHTAlpha and Newmark* classes.